sources/lib/api/gcp/bigquery/_udf.py

```diff
@@ -13,34 +13,27 @@
 """Google Cloud Platform library - BigQuery UDF Functionality."""
 
 import json
+import gcp.storage
 
 
 class FunctionCall(object):
```

Comment:
We want to keep FunctionEvaluation for local testing (I don't think we want to get rid of that; even the BQ web console uses local evaluation in the browser for UDF testing). But we don't really need FunctionCall if we expect users to only test locally or call the UDF within SQL.